com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDDefaultSettingsEditor.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine.Rendering.HighDefinition;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    [CustomEditor(typeof(HDDefaultSettings))]
+    [CanEditMultipleObjects]
+    sealed class HDDefaultSettingsEditor : Editor
+    {
+        SerializedHDDefaultSettings m_SerializedHDDefaultSettings;
+
+        internal bool largeLabelWidth = true;
+
+        void OnEnable()
+        {
+            m_SerializedHDDefaultSettings = new SerializedHDDefaultSettings(serializedObject);
+        }
+
+        public override void OnInspectorGUI()
+        {
+            var serialized = m_SerializedHDDefaultSettings;
+
+            serialized.serializedObject.Update();
```

Comment:
Is there a specific reason to update the serialized data for each frame? Sync between multiple inspectors?